### DISCUSSION TITLE
Only Shutdown ?

### DISCUSSION THREAD

**Discussion by arut16** [danieleborgo/OctoPrint-ShutdownButtonLEDBuzzer#3] [Q&A] [answered]
Hi ! 

Nice project ! Just some questions : 
- Can you give us some technical specs of differents elements ? Resistance (30.2ohm ?), LED (5mm?), buzzer ?
- Is it possible to power on thought the button after power off ? Or need to unplug/plug USB cord ?

Thanks !

**Comment by danieleborgo** [comment, marked as answer]
Hi,
this plugin support just the shutdown since it is thought to be used when you don't have a handy close to the printer OctoPrint website access.
Regarding to the resistor, it depends by how much light you want, I used a 5mm LED wit a few **thousands** ohm resistor. Remember to always check if your Raspberry pin has enough power for your LED before connecting.
Thank you
